### ISSUE TITLE
Packaging miss the package.json below

### ISSUE THREAD

**Issue by xlinbsd** [Unforgiven-wanda/electrify-updated-test#1]
Hello, thanks for maintaining electrify. Your package is not easy to find out, since it is not a simple fork.

There is a problem to complete the packaging, since the file 'package.json' is searched above the correct dirs.

For example, If my meteor application is under /my/app/path, the error is 

```
usr/local/lib/node_modules/electrify-updated-test/node_modules/electron-packager/node_modules/get-package-info/node_modules/bluebird/js/release/async.js:61
        fn = function () { throw arg; };
                           ^
Error: Cannot find module 'package.json' from '/my/app'

```

Tried on ubuntu 16.04 and OSX 10.11.6, node 4.x

**Comment by HZSamir** [comment]
Hello,
do you have a `package.json` file inside the `.electrify` folder? I'm guessing that yes, but check if the file is correct either way.

Other than that can you try packaging a fresh app? Does it work?
And can you go into more detail about how you go about packaging your app. Did you modify the `.electrify` folder?

**Comment by xlinbsd** [comment]
I confirm a `package.json` file is generated inside `.electrify`. It seems clean.

```
{
  "name": "my-electrified-app",
  "main": "index.js",
  "dependencies": {
    "electrify-updated-test": "2.1.9"
  }
}
```

I have the exact same error with a fresh _created_ meteor app.

I strictly use the following procedure to build my first uncustomized electrified apps 
`cd /path/to/my/app`
`electrify package`
_resolve on previous described crash just after_

```
INFO  electrify:electron:  packaging "my-electrified-app" for platform darwin-x64 using electron v1.4.1
x64
darwin
```

(I precise the direct `electrify run` launch correctly a desktop Electron app on OSX and often on Linux on which sometimes it starts only meteor like if I launched `meteor` command with the last usual log sentence similar to `your app is open on localhost:3000`)

**Comment by HZSamir** [comment]
Okay, try this:

Some of these issues tend to stem from bad permissions. Try resetting the permissions of your npm folder, like `sudo chown -R $USER:$GROUP ~/.npm`, then `sudo npm cache clean`

Also, go to your `.electrify folder`, delete the ``node_modules` folder. Then edit the `package.json` electrify-updated-test version to this:
`"electrify-updated-test": "2.1.10"` and reinstall.

Mac is the only platform I haven't had the chance to test this package on. I'm confident it works on both Windows and Linux, though.

**Comment by xlinbsd** [comment]
I tried your resetting procedure, exactly, on Linux and OSX.
My electrify version is now 2.1.10.

```
electrify --version
2.1.10
```

I tried on my main meteor project and on a new empty one.

No success. Same error.

I'm surprised despite updating, all new `package.json` created are pointing to
`"electrify-updated-test": "2.1.9"`

I also tested to copy the package.json on the above dir to see if the path was found. The error function is probably recursive since the error gone upper too.

I do not know how to help you help me.
Is a npm version is required ?

**Comment by HZSamir** [comment]
> Is a npm version is required ?

No, it should work whatever your version. At this point I honestly have no idea why it behaves this way. I (and some others who have tried this package) have no issues using it.

Could you upload one of the projects which gives you this error? (An empty project is fine). i'd like to take a closer look at it, and see if it works on my side.

**Comment by xlinbsd** [comment]
I can also upload an electrified version, but it is big, about 80 MB compressed (tgz -9)
[empty_test.zip](https://github.com/Unforgiven-wanda/electrify-updated-test/files/495026/empty_test.zip)

**Comment by xlinbsd** [comment]
Electrified (run) version
http://www.riegone.fr/electrified_test.tgz

**Comment by xlinbsd** [comment]
uploads done

**Comment by HZSamir** [comment]
I'll take a look at it as soon as it finishes downloading (Slow download speed here)

**Comment by HZSamir** [comment]
I'm sorry for the late answer. I've had a bit of trouble finding the source of the issue.
It should be fixed now, I've released 2.1.11. Create a new project and `electrify` then `electrify package`, it should work.

The issue stemmed from the fact that electron-packager no longer tolerates that the `.electrify/package.json` file does not specify the app version. From now on, the default app version will be "1.0.0". You can of course change that as you wish.

Hope it helps. Let me know it if works for you, and if yes, close the issue :)

**Comment by xlinbsd** [comment]
Hi,
The packaging is now working.  My apps do not open correctly but the `package` option is now working.
Thanks
